.github/workflows/ci.yml

```diff
@@ -10,31 +10,37 @@ on:
 
 jobs:
   "CI":
-    env:
-      GO111MODULE: on
-    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.platform.arch }}, ${{ matrix.platform.os }}, Go-${{ matrix.go-version }}
+    runs-on: ${{ matrix.platform.os }}
     strategy:
-      matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
-        go: ["1.20"]
-        include:
-          - os: ubuntu-latest
-            go: "1.18"
-          - os: ubuntu-latest
-            go: "1.19"
+      matrix:  # Use versions consistent with zipkin-go's Go support policy.
```

Comment:
ps after rebasing on lint, I'll put back to the other style, just change version to matrix (leave the other things alone notably that we are only testing old versions with ubuntu)